src/bin/common/parse_command.ml

```diff
@@ -894,26 +933,27 @@ let parse_output_opt =
   let docs = s_output in
 
   let interpretation =
-    let doc =
-      "Experimental support for counter-example generation. Possible \
-       values are 1, 2, or 3 to compute an interpretation before returning \
-       Unknown, before instantiation (1), or before every decision (2) or \
-       instantiation (3). A negative value (-1, -2, or -3) will disable \
-       interpretation display. Note that $(b, --max-split) limitation will \
-       be ignored in model generation phase." in
+    let doc = Format.sprintf
+        "Experimental support for counter-example generation. \
```

Comment:
The doc should not say `Experimental`: maybe something along the lines of `best effort` would better signify that it should be correct, but there is no strong guarantee on the generated model.